.github/workflows/bitrise-action.yml

```diff
@@ -1,6 +1,10 @@
 name: The bitrise action
 
-on: [push]
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [synchronize, opened, reopened, ready_for_review]
```

Comment:
Avoid duplicate triggers and allow us to test on PR events

Reply:
Nice, is synchronize only about update the stale branch?